Sources/Orbit/Foundation/Elevations/Elevation.swift

```diff
@@ -0,0 +1,194 @@
+import SwiftUI
+
+/// Use elevation to bring content closer to users.
+///
+/// Elevation levels with higher numbers are usually visually closer to the user.
+public enum Elevation {
+    case level1
+    case level2
+    case level3
+    case level4
+    case custom(opacity: Double, radius: CGFloat, x: CGFloat = 0, y: CGFloat = 0, padding: CGFloat)
```

Comment:
We can compute the padding with `radius + y` and either force it as that, or make `padding` optional and use this computation in case it is nil.

Reply:
Added.